build/bin/build-client-binary

```diff
@@ -14,9 +14,10 @@ extract_source_tarball() {
 
 gobuild_component() {
     local component=$1
-    local extract_path=${2:-"github.com/percona/$component"}
-    local component_path=$3
-    local result_file=${4:-bin/$(basename ${component_path:-$extract_path})}
+    local parent_component=${2:-$component}
```

Comment:
looks like we could do it without having one more variable.

Reply:
It's used in multiple places later